src/reducers/noteReducer.js

```diff
@@ -1,11 +1,11 @@
 const noteReducer = (state, action) => {
   switch (action.type) {
     case "ADD_NOTE":
-      return { ...state, newNotes: action.payload };
+      return { ...state, newNotes: action.payload, noteColor: "" };
```

Comment:
I'd suggest give empty string to color initially, so no need to pass it here.

Reply:
I am giving it empty so clear the value of `noteColor` when the note is added successfully.